doc/api/stream.md

```diff
@@ -494,16 +494,26 @@ added: v11.4.0
 
 Is `true` if it is safe to call [`writable.write()`][stream-write].
 
+##### writable.writableEnding
```

Comment:
The properties are in alphabetical order, so this needs to go after `writableEnded` rather than before it.